.gcloudignore

```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
+# using gcloud. It follows the same syntax as .gitignore, with the addition of
+# "#!include" directives (which insert the entries of the given .gitignore-style
+# file at that point).
+#
+# For more information, run:
+#   $ gcloud topic gcloudignore
+#
+.gcloudignore
+# If you would like to upload your .git directory, .gitignore file or files
+# from your .gitignore file, remove the corresponding line
+# below:
+.git
+.gitignore
+
+# Node.js dependencies:
+node_modules/
```

Comment:
terra-ui is an atypical app engine setup, because we're just uploading a bundle of static files, so it makes sense to use whitelisting to highlight that. Here we're following the more typical strategy of uploading a regular app from the root of the repo, so I think blacklisting is more standard.

Reply:
Reasonable, could we at least import the contents of the gitignore, and then explicitly include the docs folder?
```ignore
#!include:.gitignore
!/docs/
```